pkg/operator/sync.go

```diff
@@ -52,9 +52,12 @@ func (optr *Operator) syncAll(config *OperatorConfig) error {
 	}
 	glog.V(3).Info("Synced up all machine-api-controller components")
 
-	// In addition, if the Provider is BareMetal, then bring up
-	// the baremetal-operator pod
+	// In addition, if the Provider is BareMetal, then imay have to deploy metal3 pod.
```

Comment:
Might be better to include the condition rather than saying "may have to"
```suggestion
	// In addition, if the Provider is BareMetal and a provisioning config is available, deploy a metal3 pod.
```

Reply:
Done.